src/main/java/seedu/duke/command/misc/RemoveCommand.java

```diff
@@ -0,0 +1,69 @@
+package seedu.duke.command.misc;
+
+import seedu.duke.DukeException;
+import seedu.duke.DukeSession;
+import seedu.duke.command.ExecutableCommand;
+
+/**
+ * Represents the "remove" command.
+ */
+
+public class RemoveCommand extends ExecutableCommand {
+
+    public static int indexOfExistingIngredient;
+    String arguments;
+    String flag;
+
+    public RemoveCommand(String arguments, String flag) {
+        this.arguments = arguments;
+        this.flag = flag;
+    }
+
+    public static boolean isInList(String name) {
+        for (int i = 0; i < DukeSession.ingredients.size(); i += 1) {
+            if (DukeSession.ingredients.get(i).getName().equals(name)) {
+                indexOfExistingIngredient = i;
+                return true;
+            }
+        }
+        return false;
+    }
```

Comment:
Good abstraction of this method. I was thinking if there is any way to improve the design as the update to indexOfExistingIngredient is a side effect of this method even though it is a crucial part of this class. An alternative could be to change this method to return the index directly. An invalid index like -1 can indicate not found in list. This will also remove the need for indexOfExistingIngredient member!